metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/ExecutionProgress.java

```diff
@@ -10,28 +9,29 @@
  * @author Simon Tzanakis (Simon.Tzanakis@europeana.eu)
  * @since 2017-06-01
  */
-@Embedded
+@Entity
```

Comment:
Reverted this, to take care of the deprecation on all objects at once with another ticket.